app/[locale]/gallery/collections/(noirlab)/news-videos/[asset]/page.tsx

```diff
@@ -0,0 +1,128 @@
+import { FC } from "react";
+import { serialize } from "tinyduration";
+import { notFound } from "next/navigation";
+import { VideoObject } from "schema-dts";
+import sanitize from "sanitize-html";
+import Buttonish from "@rubin-epo/epo-react-lib/Buttonish";
+import { MediaService } from "@/services/noirlab";
+import { type Locale } from "@/lib/i18n/settings";
+import { isRubinAsset } from "@/helpers/noirlab";
+import { useTranslation } from "@/lib/i18n";
+import SingleMediaAsset from "@/components/templates/SingleMediaAsset";
+import GalleryVideo from "@/components/molecules/GalleryVideo";
+import AssetMetadata from "@/components/organisms/gallery/metadata/Asset";
+import GalleryYoutubeVideo from "@/components/molecules/GalleryYoutubeVideo";
+import SharePopup from "@/components/molecules/SharePopup";
+
+const NOIRLabVideoAsset: FC<NOIRLabAssetProps<Locale>> = async ({
+  params: { locale, asset },
+}) => {
+  const { t } = await useTranslation(locale);
+  const { data, error } = await MediaService.mediaVideosRetrieve({
+    path: { id: asset },
+    query: { lang: locale, translation_mode: "fallback" },
+    cache: "force-cache",
+  });
+
+  if (error || !data || !isRubinAsset(data.categories)) {
+    notFound();
+  }
+
+  const {
+    title,
+    description,
+    formats: { hd_1080_screen: hd1080Screen, thumb350x, videoframe },
+    release_date: dateCreated,
+    use_youtube: useYoutube = false,
+    youtube_video_id: youtubeVideoId,
+  } = data;
+
+  const credit = data.credit
+    ? sanitize(data.credit, { allowedTags: [], allowedAttributes: {} })
+    : undefined;
+  const durationAsNumber = data.duration ? parseInt(data.duration) : undefined;
+  const duration = serialize({ seconds: durationAsNumber });
+
+  const url =
+    useYoutube && youtubeVideoId
```

Comment:
some NOIRLab images are flagged to use the YouTube link, presumably for bandwidth reasons? So I'm respecting that flag and using the YouTube player